pages/common/kill.md

```diff
@@ -23,6 +23,10 @@
 
 `kill -{{9|KILL}} {{process_id}}`
 
-- Signal the operating system to pause a program, it until a SIGCONT ("continue") signal is received:
+- Signal the operating system to pause a program until a SIGCONT ("continue") signal is received:
 
 `kill -{{17|STOP}} {{process_id}}`
+
+- Send a `SIGUSR1` signal to all processes with the given GID (group id):
+
+`kill -{{SIGUSR1}} -{{group_id}}`
```

Comment:
Is the group id here numeric, or textual? If it's the latter, perhaps `group_name` would be better suited.

Reply:
It's the group ID, aka the `GID`, numerical.

Reply:
Ah, I see. Thanks for the clarification :-)